asapdiscovery-modeling/asapdiscovery/modeling/protein_prep_v2.py

```diff
@@ -78,32 +78,44 @@ def provenance(self) -> dict[str, str]:
     def cache(
         prepped_complexes: list[PreppedComplex],
         dir: Union[str, Path],
-        type=CacheType.DesignUnit,
+        # type=CacheType.DesignUnit,
     ) -> None:
         """
-        Cache a set of design units for use later.
+        Cache the list of PreppedComplex in its own folder. Each is saved as a JSON, oedu PDB and ligand SDF for vis.
         """
         dir = Path(dir)
         if not dir.exists():
             dir.mkdir(parents=True)
 
         for pc in prepped_complexes:
-            if type == CacheType.DesignUnit:
-                du_name = pc.target.target_name + ".oedu"
-                du_path = dir / du_name
-                if not du_path.exists():
-                    pc.target.to_oedu_file(du_path)
-            elif type == CacheType.JSON:
-                json_name = pc.target.target_name + ".json"
-                json_path = dir / json_name
-                if not json_path.exists():
-                    pc.to_json_file(json_path)
+            # create a folder for the complex data
+            complex_folder = dir.joinpath(pc.target.target_name)
```

Comment:
We may need to be careful here about targets with the same name.

Reply:
Good point we could change this to `YYYY-MM-DD-target-name`?

Reply:
Or is there a unique reproducible hash we could use with the target name maybe sha256 of the pdb file, I guess this depends under what conditions are two prepared complexes regarded the same?

Reply:
The current method in [8c5e275](https://github.com/choderalab/asapdiscovery/pull/608/commits/8c5e27590ab870e639d8703a878dd2ef291af298) uses a hash based on the input target to check for matches in the cache.